storybook/src/components/Page/Page.docs.mdx

```diff
@@ -0,0 +1,33 @@
+{/* @license CC0-1.0 */}
+
+import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as PageStories from "./Page.stories.tsx";
+import README from "../../../../packages/css/src/components/page/README.md?raw";
+
+<Meta of={PageStories} />
+
+<Markdown>{README}</Markdown>
+
+## Example
+
+There’s not much to see in the example.
+In a wide window, or when zooming out, you’ll notice that the width of the pink box gets limited by the maximum width of the Page container.
+
+<Primary />
```

Comment:
The height of this story is 100dvh now, which looks a little odd imo. Do we want to limit that height?

Reply:
I added an override to our Storybook styles 🙈